src/binary/int.rs

```diff
@@ -55,6 +56,26 @@ impl Int {
         })
     }
 
```

Comment:
In this file:
* Adding write support for [the low-level encoding primitive `Int`](https://amzn.github.io/ion-docs/docs/binary.html#uint-and-int-fields), not to be confused with the Ion `integer` type.